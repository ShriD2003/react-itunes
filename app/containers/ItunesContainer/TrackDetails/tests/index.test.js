/**
 *
 * Tests for TrackDetails container
 *
 *
 */

import React from 'react';
// import { fireEvent } from '@testing-library/dom';
import { renderProvider } from '@utils/testUtils';
import { TrackDetailsTest as TrackDetails } from '../index';

describe('TrackDetails SubContainer tests', () => {
  let submitSpy;

  beforeEach(() => {
    submitSpy = jest.fn();
    submitSpyClearTrackDetails = jest.fn();
  });

  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(
      <TrackDetails dispatchClearTrackDetails={submitSpyClearTrackDetails} dispatchTrackSearch={submitSpy} />
    );
    expect(baseElement).toMatchSnapshot();
  });
});
