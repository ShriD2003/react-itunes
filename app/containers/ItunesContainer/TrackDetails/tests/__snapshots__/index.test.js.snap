// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackDetails SubContainer tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-braxZu hWChFe"
      minwidth="10%"
    >
      <div
        class="ant-skeleton ant-skeleton-active css-dev-only-do-not-override-1pg9a38"
      >
        <div
          class="ant-skeleton-content"
        >
          <h3
            class="ant-skeleton-title"
            style="width: 38%;"
          />
          <ul
            class="ant-skeleton-paragraph"
          >
            <li />
            <li />
            <li
              style="width: 61%;"
            />
          </ul>
        </div>
      </div>
    </div>
  </div>
</body>
`;
